client/hawc_client/__init__.py

```diff
@@ -511,6 +511,51 @@ def visual_list(self, assessment_id: int) -> pd.DataFrame:
         return pd.DataFrame(response_json)
 
 
+class StudyClient(BaseClient):
+    """
+    Client class for study requests.
+    """
+
+    def create(
+        self, reference_id: int, short_citation: str, full_citation: str, data: Dict = {}
```

Comment:
this is a common python issue. this creates a single dict that is reused anywhere a default  is not provided; so if it's modified it persists even after the method call ends.  To fix:

```python
def create(self, data: Optional[Dict] = None):
    if data is None:
         data = {}
```

Reply:
Good to know, thanks! Fixed.